### PR TITLE
Fixed expand/collapse shortcut to not conflict with browser shortcuts

### PR DESCRIPTION
**Pull request by alex-ozdemir**
Allows both `T` and `t`.

It had been [Shift]+[+] before.

In response to #33791.

cc @Manishearth 
r? @GuillaumeGomez

### CODE REVIEW

File: src/librustdoc/html/static/main.js [hunk 1]
```diff
@@ -124,7 +124,8 @@
             focusSearchBar();
             break;
 
-        case "+":
+        case "t":
```
Comment (GuillaumeGomez) [inline src/librustdoc/html/static/main.js]: If you're writing into the search field, does it expand all sections or not?

**Comment by GuillaumeGomez** [comment]
Also, I open a new tab with "CTRL+T". Does it expand all sections?

**Comment by Manishearth** [comment]
Can't be t, since its already a well known shortcut.

How about e or slash?

**Comment by alex-ozdemir** [comment]
Copying one of @GuillaumeGomez's comments here:

> Adding another key to the "+" I guess? Something like "SHIFT+'+'" maybe?

One thing to note - it currently is [Shift] + [+=] (rather than just [+=]). If we like that we can leave it.

Otherwise any of "e", [?/], or [|\] seem fine to me, although [Shift] + [?/] brings up the help screen.

**Comment by ollie27** [comment]
[Ctrl]+[T] isn't a problem because of [these lines](https://github.com/rust-lang/rust/blob/ab09fbca234d491fbd09857eb414d2586d283c67/src/librustdoc/html/static/main.js#L104-L106). Neither is [Ctrl]+[+] for that matter.

The shortcut doesn't work properly at the moment anyway because it's missing `ev.preventDefault();`.

**Comment by alex-ozdemir** [comment]
Thanks! I was only testing on Chrome, so I didn't realize the `ev.preventDefault()` was necessary.

**Comment by Manishearth** [comment]
Ah, okay. Could you upload a copy of this running somewhere?

**Comment by GuillaumeGomez** [comment]
> Ah, okay. Could you upload a copy of this running somewhere?

I'd like to test it as well.

**Comment by alex-ozdemir** [comment]
Good idea.

The documentation for `std` built with the new version of `rustdoc` is [here](http://alex-ozdemir.github.io/test/doc/std/).

**Comment by Manishearth** [comment]
looks good to me

@bors delegate=GuillaumeGomez

**Comment by GuillaumeGomez** [comment]
Good for me as well. Thanks @alex-ozdemir!

@bors: r+ rollup

**Pull request merged**